lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml

```diff
@@ -12,6 +12,36 @@ all_interfaces:
   get_context: ~
   get_value: '/^interface (.*)$/'
 
+global_ingress_replication_bgp:
+  _exclude: [N3k-F, N5k, N6k, N7k, N9k-F]
+  os_version: 'N9k:9.2'
+  kind: boolean
+  get_value: '/^global ingress-replication protocol bgp$/'
+  set_value: '<state> global ingress-replication protocol bgp'
+  default_value: false
+
+global_mcast_group_l2:
+  _exclude: [N3k-F, N5k, N6k, N7k]
+  os_version: 'N9k, N9k-F:9.2'
```

Comment:
I don't understand why this property is different for the `_exclude` and `os_version`

Reply:
The exclude option says that this property is not applicable for N3k-F, N5k, N6k and N7k. It is applicable to N9k and N9k-F, however, only if this platform is running version 9.2 or above. The os_version is for checking the minimum version capability.

Reply:
Good point. 👍

Reply:
That I understand.  The question I have is... Is this property supported on N9k running versions older then `9.2`?

Reply:
No. only from 9.2

Reply:
I don't remember the way the `os_version` string lookup works but I thought that you had to specify a version string for each platform.

`os_version: 'N9k:9.2, N9k-F:9.2`

Reply:
Not really. If the version is different for different platforms, then what you say is correct. If the version is same for multiple platforms, you can use comma notation.
Check this:
https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/cmd_ref/vpc.yaml#L98